### ISSUE TITLE
refreshAccessToken() returns DbxRefreshResult with incorrect expiresIn

### ISSUE THREAD

**Issue by skand888** [dropbox/dropbox-sdk-java#294]
refreshAccessToken() returns DbxRefreshResult with getExpiresAt() with a date far in the future. This happens because in passes credential.getExpiresAt() as expiresIn parameter to DbxRefreshResult  constructor.

```
        @Override
        public DbxRefreshResult refreshAccessToken() throws DbxException {
            credential.refresh(this.getRequestConfig());
            return new DbxRefreshResult(credential.getAccessToken(), **credential.getExpiresAt()**);
        }
```
```
    public DbxRefreshResult(/*@NotNull*/String accessToken, long **expiresIn**, String scope) {
        if (accessToken == null) {
            throw new IllegalArgumentException("access token can't be null.");
        }
        this.accessToken = accessToken;
        this.expiresIn = expiresIn;
        this.issueTime = System.currentTimeMillis();
        this.scope = scope;
    }
```
As the result, current epoch milliseconds are added twice.

**Comment by greg-db** [comment]
Thanks for the report! I'll ask the team to get that fixed up.

**Comment by PhilippC** [comment]
any update on this?

**Comment by tahsini** [comment]
Hi, no update on this yet.

**Comment by greg-db** [comment]
This is fixed in release [v4.0.1](https://github.com/dropbox/dropbox-sdk-java/releases/tag/v4.0.1).
